src/game_event.cpp

```diff
@@ -521,11 +521,9 @@ void Game_Event::Update() {
 		return;
 	}
 
-	if (IsProcessed()) {
-		return;
-	}
-	SetProcessed(true);
-
+	// RPG_RT runs the parallel interpreter everytime Update is called.
+	// That means if the event updates multiple times due to makeway,
+	// the interpreter will run multiple times per frame.
```

Comment:
> Run parallel events everytime update is called 

Could you also add a comment that this also emulates a bug because the Wait count also decrements

Reply:
The wait count decrementing is a result of running the interpreter again. Do you still want a special comment for that?

Reply:
because the wait is in frames and not in interpreter invocations I don't find this intuitive so yes, explicitly mentioning this is better.

Note, old comment was:
> This results in waits to finish quicker when an event collides with this event and
> emulates a RPG Maker bug)

Reply:
Added